### DISCUSSION TITLE
Change Campus email domain

### DISCUSSION THREAD

**Discussion by basilgass** [community/community#61849] [GitHub Education]
### Hi everyone,

I wanted to join the Global Campus as a teacher. When I enter the school name, it founds it... But the domain has been changed for this school 3 years ago!

I saw this discussion (https://github.com/orgs/community/discussions/55619) and it seems that I won't be able to change this on my own.

So my question or request: 

- can you change the domain name ? I can provide more information if you want for my school.
- the discussion mentioned above mentioned a ticket... Do I need to open one ? And if so, how ?

Thanks for your help !

**Comment by moonfury-ops** [comment]
Thank you
